modular_skyrat/modules/organs/code/stomach.dm

```diff
@@ -0,0 +1,7 @@
+/obj/item/organ/stomach/oversized
```

Comment:
how does this work with other race stomachs (e.g. ethereal)?

Reply:
It doesn't, as I don't think there's an easy way to effect the stomach's metabolism without giving them a new one.

Reply:
Thing is, this'll fuck with ethereals, IPCs, and skeleton stomachs

Reply:
Handled!